docs/general/ledger.md

```diff
@@ -176,5 +185,54 @@ extrinsics are available in the light version. The following
 [repository by Zondax](https://github.com/Zondax/ledger-polkadot) lists the currently supported
 Democracy extrinsics on the full ledger.
 
+## Ledger Developer Release
+
+:::warning
+
+These instructions are for development installation only. It is recommended to install the
+application from Ledger Live unless you _know exactly what you're doing_.
+
+:::
+
+### Why you might need the Developer Release
+
+Ledger apps for the Polkadot and Kusama ecosystems are developed by [Zondax](https://zondax.ch/).
+The developer release of the apps are published before updates are usually implemented on the
+runtime to make sure that when the new runtime version is deployed the app can be downloaded and
+installed using [Ledger Live](https://www.ledger.com/ledger-live). However, because it takes some
+time for Ledger to audit and review the release, there might be a risk that the app upgrade is not
+available on Leger Live when the runtime is deployed. If this happens users cannot use Ledger
+devices together with the Polkadot-JS UI, and most likely while signing for transaction they will
+incur in the error message "txn version not supported". Please do not panic if this happens as there
+are solutions to this problem. If you cannot wait a couple of days that the app passes the Ledger
+audit, you can install the developer release from the shell using the latest version published on
```

Comment:
Zondax may update the apps even when there is no runtime upgrade on the network (like some of the backlogged feature requests). Rephrased the content a bit to reflect that.
Please accept this suggestion if the rephrased content captures the original intent.

```suggestion
Ledger apps for the Polkadot and Kusama ecosystems are developed by [Zondax](https://zondax.ch/).
When new functionalities are added to the Ledger apps, they are made available on a developer release for testing purposes. After a successful audit and review, the apps would be available for download and
installation using [Ledger Live](https://www.ledger.com/ledger-live). As it takes some
time for Ledger to audit and review the release, the app upgrade option may not be 
available on Ledger Live when the new runtime is deployed on the network. If this happens, users cannot use Ledger
devices with the Polkadot-JS UI, and while signing for a transaction, they will most likely
incur the error message "txn version not supported". Please do not panic if this happens, as there
are solutions to this problem. If you cannot wait a couple of days until the app passes the Ledger
audit, you can install the developer release from the shell using the latest version published on
```